src/utils/patchVotes.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+
+export const patchVotes = (article_id) => {
```

Comment:
Do you want a way of recucing the votes too? This is a consideration, and an implementation choice, I'm just curious